packages/lesswrong/components/posts/PostsPage/PostBody.tsx

```diff
@@ -1,9 +1,14 @@
-import React from 'react';
+import React, { useRef } from 'react';
 import { Components, registerComponent } from '../../../lib/vulcan-lib';
 import { nofollowKarmaThreshold } from '../../../lib/publicSettings';
 import { useSingle } from '../../../lib/crud/withSingle';
 import mapValues from 'lodash/mapValues';
 import type { SideCommentMode } from '../../dropdowns/posts/SetSideCommentVisibility';
+import { useVote } from '../../votes/withVote';
+import { getVotingSystemByName } from '../../../lib/voting/votingSystems';
+import type { ContentItemBody, ContentReplacedSubstringComponent } from '../../common/ContentItemBody';
+
+const enableInlineReactsOnPosts = true; //TODO
```

Comment:
Hm, not sure how that showed up on the wrong side, but see https://github.com/ForumMagnum/ForumMagnum/pull/7933/files#r1357191063